java/ql/src/experimental/Security/CWE/CWE-094/InjectionLib.qll

```diff
@@ -0,0 +1,14 @@
+import java
+import semmle.code.java.dataflow.FlowSources
+
+/**
+ * Holds if `fromNode` to `toNode` is a dataflow step that returns data from
+ * a bean by calling one of its getters.
+ */
+predicate returnsDataFromBean(DataFlow::Node fromNode, DataFlow::Node toNode) {
```

Comment:
This doesn't seem to have anything to do with beans, it's just any old getter. Rename `hasGetterFlow`?

Reply:
Well, the predicate doesn't fully describe a Java bean but I meant a call to a getter of a bean. No problem, I renamed it to `hasGetterFlow`.